fv-swiftype.php

```diff
@@ -758,20 +768,23 @@ function post_thumbnail_html( $html, $post_id, $post_thumbnail_id, $size ) {
     global $post;
     if( empty($post->fv_swiftype_featured) ) return $html;
     
-    $width = get_option('thumbnail_size_w');
-    $height = get_option('thumbnail_size_h');
-    
-    global $_wp_additional_image_sizes;
-    if( isset($_wp_additional_image_sizes[$size]) ) {
-      $width = $_wp_additional_image_sizes[$size]['width'];
-      $height = $_wp_additional_image_sizes[$size]['height']; 
-    }
-    
-    $width = intval($width);
-    $height = intval($height);
-    
-    if( $width > 0 && $height > 0 ) {
-      $style = ' style="max-width: '.$width.'px; max-height: '.$height.'px"';
+    $style = '';
+    if( !isset( $this->aOptions['disable_image_style'] ) || !$this->aOptions['disable_image_style'] ) {
+      $width = get_option( $size.'_size_w');
+      $height = get_option( $size.'_size_h');
```

Comment:
Adding size variables from function call to get right size ( not only thumbnail)